homeassistant/components/xiaomi_aqara/const.py

```diff
@@ -13,3 +13,43 @@
 CONF_SID = "sid"
 
 DEFAULT_DISCOVERY_RETRY = 5
+
+BATTERY_MODELS = [
```

Comment:
Can we put this in `PyXiaomiGateway` instead ?

Reply:
After digging though, I realize that all the models are in lots of the other platforms as well.  It would be nice if `PyXiaomiGateway` took care of this detail.

Reply:
Yes, all the models are handled in the integration. I'll try to find a time to do a refactor and move those to `PyXiaomiGateway`.